auth-foundation/src/main/java/com/okta/authfoundation/AuthFoundationDefaults.kt

```diff
@@ -72,7 +71,7 @@ object AuthFoundationDefaults {
 
     /** The default [CookieJar]. By default, it adds a DT cookie for identifying the device.
      * To use the default [CookieJar] in OkHttp, set this to [CookieJar.NO_COOKIES] */
-    var cookieJar: CookieJar by NoSetAfterGetWithLazyDefaultFactory { DeviceTokenCookieJar(clock) }
+    var cookieJar: CookieJar by NoSetAfterGetWithLazyDefaultFactory { CookieJar.NO_COOKIES }
```

Comment:
How do you propose handling DeviceToken support for IDX? The documentation for this property no longer matches up with the implementation, so I'm wondering what the plan is?

Furthermore, if the cookieJar is set to NO_COOKIES, is this property even needed anymore?

Reply:
I decided to revert this, and instead fixed the issues with the following:
1. Lazily loading DT cookie in DeviceTokenCookieJar to avoid crashes when users create OidcConfiguration in a Jetpack startup initializer.
2. Attempting to reinitialize DeviceTokenProvider's storage by deleting the storage during Exception, and then recreating the storage.

These should fix most of the issues reported by users. The only exception being if the encryption key is itself corrupted, which can happen and has been reported as an issue for EncryptedSharedPreferences. But, that case should be much rarer than the issues addressed by the above two fixes.